ggml-cuda.cu

```diff
@@ -116,6 +116,8 @@
 #include "ggml.h"
 #include "ggml-backend-impl.h"
 
+#define CUDART_CI     11070 // CUDA 11.7, version used for the Github CI
```

Comment:
I would prefer a name that explains how this is meant to be used, such as `CUDART_HMAX` or similar.

Reply:
Tagging @henk717 , if we want to use a different version of CUDA toolkit should we be changing this value?

Reply:
> if we want to use a different version of CUDA toolkit should we be changing this value?

The value should only be changed if you find a lower version of CUDA that supports __hmax2 on Pascal. In which case, you should make a PR.